### PR TITLE
Fix a small typo

### PR DESCRIPTION
**Pull request by DanielNoord**
Just spotted this minor typo.
Great work on the project and video! I enjoyed it a lot!

Edit: I just saw that the video is almost 10 months old.. Well thank the YouTube algorithm for showing it to me now!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 - Full transparency + translucency support
 - Sprite blocks (flowers)
 - Animated blocks (water + lava)
-- Distaince fog
```
Comment (Deniz073) [inline README.md:17]: indeed a typo

**Comment by equinox** [comment]
It'd be neat if you could also fix line 12's "colision" => "collision" too (probably not worth a separate PR so just gonna comment here, sorry!).

**Comment by DanielNoord** [comment]
@jdah probably wants to merge these commits now as to not pollute the commit history too much.

**Comment by jdah** [comment]
Will merge, looks like I typed the description a bit fast haha
Thanks for catching these!

**Pull request merged**